### ISSUE TITLE
RAM Management

### ISSUE THREAD

**Issue by nh1402** [sfdroid/sfdroid-misc#15]
All Android services are running, and is unrestricted, we should minimise Android OS RAM usage as much as possible, maybe give it a size limit?

**Comment by ghost** [comment]
good idea. On 1GB ram very low ram left :/ Maybe remove unnecessary apps/services?

**Comment by makayama** [comment]
i get 797mb of memory in the first use for sfdroid
